### DISCUSSION TITLE
Billing only working for first plan

### DISCUSSION THREAD

**Discussion by sabinshrestha8** [Kyon147/laravel-shopify#27] [General]
By configuring this package, I have created 2 pricing plans (Monthly & Annual) using PlanSeeder and followed the steps as documented. Plan 1 is successfully redirecting to billing page at the time of purchase while Plan 2 is not redirecting to the billing page. can this be any kind of bug or is it problem with my codebase??

Hope to get a reply ASAP

**Comment by Kyon147** [comment]
Hi @sabinshrestha8 

Did you get this working in the end? There have been some updates to the package over the last couple months, so please try using the latest version.

Good luck!
